### PR TITLE
chore: use replace some %u with LV_PRIu32

### PR DESCRIPTION
**Pull request by W-Mai**
Help us review this PR! Anyone can [approve it or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### Description of the feature or fix

use replace some %u with LV_PRIu32

### Checkpoints
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html)

### CODE REVIEW

**Comment by W-Mai** [comment]
image

It seems that CI meets some weird problem.

**Comment by PGNetHun** [comment]
> image
> It seems that CI meets some weird problem.

yes, it must be only a temporary problem in github services….

File: src/libs/freetype/lv_freetype_glyph.c [hunk 1]
```diff
@@ -101,7 +101,7 @@ static bool freetype_get_glyph_dsc_cb(const lv_font_t * font, lv_font_glyph_dsc_
 
     lv_cache_entry_t * entry = lv_cache_acquire_or_create(glyph_cache, &search_key, dsc);
     if(entry == NULL) {
-        LV_LOG_ERROR("glyph lookup failed for unicode = %u", unicode_letter);
+        LV_LOG_ERROR("glyph lookup failed for unicode = %" LV_PRIu32, unicode_letter);
```
Comment (FASTSHIFT) [inline src/libs/freetype/lv_freetype_glyph.c]: LV_PRIx32

File: src/libs/lodepng/lv_lodepng.c [hunk 2]
```diff
@@ -167,7 +167,7 @@ static lv_result_t decoder_open(lv_image_decoder_t * decoder, lv_image_decoder_d
                 if(png_data != NULL) {
                     lv_free((void *)png_data);
                 }
-                LV_LOG_WARN("error %u: %s\n", error, lodepng_error_text(error));
+                LV_LOG_WARN("error %" LV_PRIu32 ": %s\n", error, lodepng_error_text(error));
```
Comment (FASTSHIFT) [inline src/libs/lodepng/lv_lodepng.c]: LV_LOG_WARN("error %u: %s", error, lodepng_error_text(error));
'error' is a 'unsigned' type, using '%u' printing is reasonable.

File: src/libs/freetype/lv_freetype_glyph.c (hunk 1 above)
Comment (XuNeo) [reply src/libs/freetype/lv_freetype_glyph.c]: 0x %"LV_PRIx32"

**Review by FASTSHIFT — APPROVED** [review]

**Pull request merged**